core/app/models/spree/distributed_amounts_handler.rb

```diff
@@ -1,17 +1,16 @@
 module Spree
   class DistributedAmountsHandler
-    attr_reader :line_item, :order, :promotion, :total_amount
+    attr_reader :distributed_amounts, :line_items, :total_amount
```

Comment:
I think we don't need this `distributed_amounts`  reader method here

Reply:
👍 thanks